docs/maps/index.asciidoc

```diff
@@ -12,16 +12,45 @@ With *Elastic Maps*, you can:
 * Create maps with multiple layers and indices.
 * Upload GeoJSON files into Elasticsearch.
 * Embed your map in Dashboards.
-* Plot individual documents or use aggregations to plot any data set, no matter how large.
 * Create choropleth maps.
-* Use data driven styling to symbolize features from property values.
+* Use data driven styling to symbolize features from data values.
 * Focus the data you want to display with searches.
 
 Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.
 
 [role="screenshot"]
 image::maps/images/sample_data_ecommerce.png[]
 
+[float]
+=== Create maps with multiple layers and indices
+Context is a force multiplier for spatial analysis, revealing how and where data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions.
+Create maps with multiple layers and indices to show all of your data in a single map. Plot individual documents or use aggregations to plot any data set, no matter how large.
```

Comment:
"context is a force multiplier for spatial analysis" is difficult to parse.  Is that content needed?   

You can use multiple layers and indices to show all your data in a single map.  This enables your map to show how data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions.  You can plot individual documents or use aggregations to play any data set, no matter how large.